app/components/mig/login/login.js

```diff
@@ -1,5 +1,5 @@
 /*global HTMLElement */
-import closest from './closest'
+import closest from '../../../utils/closest'
```

Comment:
Yep. Everything in `/utils` will get extracted into their own packages. Leaving them in for now to be more battle tested (slightly less overhead when refactoring).